polaris/dataset/converters/_pdb.py

```diff
@@ -169,7 +169,7 @@ def _convert_pdb(
             dtype = DTYPE_DICT.get(col_name, col_val.dtype)
             group.array(col_name, data=col_val, dtype=dtype)
 
-    def convert(self, path: Union[str, List[str]], factory: "DatasetFactory") -> FactoryProduct:
+    def convert(self, path: Union[str, List[str]], factory: "DatasetFactory", **kwargs) -> FactoryProduct:
```

Comment:
**Suggestion:** It seems to me that appending is also relevant for the PDB converter. Maybe we can introduce functionality in the super class that can be shared between the different converters.